yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm

```diff
@@ -337,6 +337,11 @@
 		target.visible_message(span_big("[target] looks to have experienced a revelation!"), \
 							   span_warning("False faces all d<b>ark not real not real not--</b>"))
 		target.setOxyLoss(0) //In case the shadowling was choking them out
+		if(iscarbon(target))
```

Comment:
this is going to let people know who thralls are (you accidentally resolved this)

Reply:
if i was a sling i'd rather them keep glowy and just not interact with them honestly